mongothon/schema.py

```diff
@@ -5,9 +5,21 @@
 class Schema(schemer.Schema):
     """A Schema encapsulates the structure and constraints of a Mongo document."""
 
+    indexes = []
+
     def __init__(self, doc_spec, **kwargs):
-        super(Schema, self).__init__(doc_spec, **kwargs)
+        super(Schema, self).__init__(doc_spec, **{k:v for k,v in kwargs.iteritems() if k != 'indexes'})
```

Comment:
aware this is hackish... is there a better way to do it? I want to pass an extra `kwarg` here that the underlying class doesn't support.